sdcm/fill_db_data.py

```diff
@@ -2908,7 +2915,7 @@ def run_db_queries(self, session, default_fetch_size):
         # pylint: disable=too-many-branches,too-many-nested-blocks
 
         for item in self.all_verification_items:
-            session.execute("USE keyspace1;")
+            session.execute("USE keyspace_fill_db_data;")
```

Comment:
`run_db_queries` is run from `verify_db_data` and the keyspace has already been set there, why to set it again?

Reply:
Removed.

Reply:
@bentsi 

Some queries contains statement of switch keyspace, reset keyspace at the beginning.
I just added the `session.set_keyspace("keyspace_fill_db_data")` back.

Otherwise, we will see error:
```
InvalidRequest: Error from server: code=2200 [Invalid query] message="unconfigured table undefined_column_handling_test"
```